L1Trigger/TrackFindingTracklet/test/AnalyzerTBout.cc

```diff
@@ -0,0 +1,432 @@
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+#include "FWCore/Framework/interface/Run.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/Utilities/interface/Exception.h"
+#include "CommonTools/UtilAlgos/interface/TFileService.h"
+#include "DataFormats/Common/interface/Handle.h"
+
+#include "SimTracker/TrackTriggerAssociation/interface/StubAssociation.h"
+#include "L1Trigger/TrackTrigger/interface/Setup.h"
+#include "L1Trigger/TrackerTFP/interface/DataFormats.h"
+#include "L1Trigger/TrackFindingTracklet/interface/ChannelAssignment.h"
+#include "L1Trigger/TrackFindingTracklet/interface/Settings.h"
+
+#include <TProfile.h>
+#include <TProfile2D.h>
+#include <TH1F.h>
+#include <TH2F.h>
+
+#include <vector>
+#include <deque>
+#include <set>
+#include <cmath>
+#include <numeric>
+#include <sstream>
+
+using namespace std;
+using namespace edm;
+using namespace trackerTFP;
+using namespace tt;
+
+namespace trklet {
+
+  // stub resolution plots helper
+  enum Resolution { Phi, Z, NumResolution };
+  constexpr initializer_list<Resolution> AllResolution = {Phi, Z};
+  constexpr auto NameResolution = {"Phi", "Z"};
+  inline string name(Resolution r) { return string(*(NameResolution.begin() + r)); }
+
+  /*! \class  trklet::AnalyzerTBout
+   *  \brief  Class to analyze hardware like structured TTStub Collection generated by Tracklet
```

Comment:
This comment explaining what the class sounds mysleading "analyzer HW". Isn't it just checking the performance of the TBoutProducer?

Reply:
yes